<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysOverviewObjectTriggerDSLEvaluatorDialog</class>
 <widget class="QWidget" name="SysOverviewObjectTriggerDSLEvaluatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="msgIDLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Message ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MsgIDLineEdit" name="msgIDLE" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="msgCodeLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Messag Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MsgCodeLineEdit" name="msgCodeLE" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="formatStringLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Format String</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="formatStringPlainTextEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="formatterBtn">
         <property name="text">
          <string>Formatter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MsgIDLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">MsgIDLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MsgCodeLineEdit</class>
   <extends>QWidget</extends>
   <header location="global">MsgCodeLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
